src/ray/raylet/local_object_manager.cc

```diff
@@ -271,44 +281,83 @@ void LocalObjectManager::AddSpilledUrls(
     const auto node_id_object_spilled =
         is_external_storage_type_fs_ ? self_node_id_ : NodeID::Nil();
 
+    const auto unpin_callback = [this, object_id, object_url, callback,
```

Comment:
Can we just make it a private method?

Reply:
Something like `HandleObjectUnpinned` or something

Reply:
Any particular reason why? Defining these kind of callbacks as inline lambdas is definitely the most common pattern in Ray.

Reply:
Hmm I am not sure if this kind of callback is common (use callback when we can have a private method). I prefer private method because if there’s some fault, stack trace will look better

Reply:
That's a good reason, I'll change this to a `std::bind`ed private method.